package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.22",
+  "version": "1.0.23",
```

Bringing such feature should be a minor, not a patch of me ou follow semver.

So IG should have been a

```suggestion
  "version": "1.1.0",
```

---

Yeah but I just didn't follow semver from the beginning. I've added the image support later and it should've been v1.1.0 and now with the video should've been v1.2.0. But it doesn't really important I guess. I'm just trying out things. Not sure this video feature is going to work for everyone.

---

OK, thanks for replying me